docker-compose.yml

```diff
@@ -2,13 +2,15 @@ version: '2.4'
 
 services:
   nextcloud-ephemeral-master:
+    environment:
+      - PORT
```

Comment:
This does nothing. The `environment` key is to pass into the container, not to use in the yaml.

Reply:
Removed